dlc/src/main/scala/org/bitcoins/dlc/BinaryOutcomeDLCClient.scala

```diff
@@ -1077,8 +1089,8 @@ object BinaryOutcomeDLCClient {
       remoteInput = offer.totalCollateral,
       fundingUtxos = fundingUtxos,
       remoteFundingInputs = offer.fundingInputs,
-      winPayout = offer.contractInfo.values.head,
-      losePayout = offer.contractInfo.values.last,
+      winPayout = offer.contractInfo.values.last,
+      losePayout = offer.contractInfo.values.head,
```

Comment:
This doesn't seem right, we are doing the following for the corresponding hashes

```
outcomeWin = offer.contractInfo.keys.head,
outcomeLose = offer.contractInfo.keys.last,
```

Reply:
Right but `contractInfo` contains the payouts of the counter-party